rasa/core/channels/webexteams.py

```diff
@@ -26,21 +26,21 @@ def __init__(self, access_token, room):
     ) -> None:
         recipient = self.room or recipient_id
         for message_part in text.split("\n\n"):
-            self.api.messages.create(roomId=recipient, text=message_part)
-
+            self.api.messages.create(roomid=recipient, text=message_part) if self.room else self.api.messages.create(toPersonId=recipient, text=message_part)
```

Comment:
We could change this to an `if...else` so that it's easier to read:
```python
            if self.room:
                self.api.messages.create(roomid=recipient, text=message_part)
            else:
                self.api.messages.create(toPersonId=recipient, text=message_part)
```